I.write-tests/01-greet-people/greet-people.js

```diff
@@ -1,18 +1,26 @@
-function greetPeople(people) {
+  function greetPeople(people) {
     var greeting = 'Hello ';
 
-    people.forEach(function (person) {
-        greeting = greeting + person;
-    });
-
-    return greeting;
-}
+    people.forEach(function (person, index) {
+    if(index!== people.length-1){
+        greeting =  greeting + person + ", ";
 
+      }else{
```

Comment:
This `else` block seems a bit odd. It is not well formatted which makes it difficult to follow.
Can we try formatting it so it becomes more readable?
If you are using VSCode, the editor should be able to do that for you

Reply:
We can make it like this
function greetPeople(people) {
  var greeting = "Hello ";

  people.forEach(function(person) {
    greeting = greeting + person + ", ";
  });

  return greeting.slice(0, -2);
}

Reply:
is it better?